pkgs/top-level/all-packages.nix

```
@@ -33988,8 +33988,7 @@ with pkgs;
 
   nerd-font-patcher = callPackage ../applications/misc/nerd-font-patcher { };
 
-  netmaker = callPackage ../applications/networking/netmaker {subPackages = ["."];};
-  netmaker-full = callPackage ../applications/networking/netmaker { };
+  netmaker-full = callPackage ../by-name/ne/netmaker/package.nix { withServer = true; withCLI = true; };
```

```suggestion
  netmaker-full = netmaker.override { withServer = true; withCLI = true; };
```

It's probably better to use override instead of re-importing the package, I think. (Same for the other cases of this.)

---

yeah, that `by-name` validation seems to be a lot happier about override all around